RELEASE/scripts/autoscend.ash

```diff
@@ -2487,6 +2487,35 @@ void resetState() {
 	}
 }
 
+boolean process_tasks()
+{
+	string [string,int,string] task_order;
+	if(!file_to_map("autoscend_task_order.txt", task_order))
+	{
+		abort("Could not load /data/autoscend_task_order.txt");
+	}
+
+	string task_path = my_path();
```

Comment:
will this cause issues with paths who have space in the name when we try to place them in the data file?

for example:
```
> ash my_path()

Returned: Grey Goo
```
the data file is seperated by tabs right, so would it handle spaces as a single entity?

Reply:
might be simpler to change it to an int and use my_path_id( ) instead of my_path() and change default to 0.

Reply:
That sounds like a good idea. could then include a comment in each of the data files indicating which id number is which path
eg
`## ID 0 = default`
at the top of the file

Reply:
This should be fine with paths with spaces in the names because the data files use tabs to separate.